com.microsoft.mrtk.windowsspeech/MRTK.Speech.Windows.asmdef

```diff
@@ -3,6 +3,7 @@
     "rootNamespace": "Microsoft.MixedReality.Toolkit.Speech.Windows",
     "references": [
         "Microsoft.MixedReality.Toolkit.Core",
+        "Microsoft.MixedReality.Toolkit.Input",
```

Comment:
This seems weird. (I don't think we should move these base classes, but if they _need_ to be moved) I'm not sure why the speech classes weren't moved into the speech package. That's where the concrete implementations already live.